CHANGES_NEXT_RELEASE

```diff
@@ -4,3 +4,4 @@
        the new CLI parameter -mutexTimeStat is set (Issue #911)
 - Fix: Removing trailing slashes for path of URI before treating the request (Issue #828)
 - Fix: Fixed a bug regarding subscriptions with empty entity-type (Issue #691)
+- Fix: Fixed a bug regarding Service paths and DELETE operation (Issue # 885)
```

Comment:
The bug actually is broader, so probably this entrey would be more accurate:

"Fix: updateContext propragation to Service Path subtree, instead of just scoping to the particular Service Path in the Fiware-Service header (Issue # 885)"

Or a improved English wording :)